x-pack/plugins/cases/server/client/configure/create_mappings.ts

```diff
@@ -41,7 +41,7 @@ export const createMappings = async (
       refresh,
     });
 
-    return theMapping.attributes.mappings;
+    return theMapping.attributes.mappings as ConnectorMappingsAttributes[];
```

Comment:
I think that if we define the return type of the functions and cast in `ConnectorMappingsService` we can avoid the casting here. Same for the all functions of the `configure` folder.